docs/submission/submitting-molecular-data.md

```diff
@@ -3,15 +3,13 @@ id: submitting-molecular-data
 title: Submitting Molecular Data
 platform_key: DOCS_SUBMITTING_MOLECULAR_DATA
 ---
-
-![Reminder Banner](/assets/submission/banner-reminder.svg)
-
-Molecular data consists of raw data files (e.g. sequencing reads), as well as any associated file metadata (data that describes your data).
+Molecular data consists of raw molecular data files (e.g. sequencing reads), as well as any associated file metadata (data that describes your data).
 
 Raw molecular data is submitted to a **Regional Data Processing Centre (RDPC)**. RDPCs are responsible for processing your program's molecular data according to the [Analysis Pipeline](/docs/analysis-workflows/analysis-overview). If you are unsure which RDPC you should submit to, please [contact the DCC](https://platform.icgc-argo.org/contact).
 
-> [Sample Registration](/docs/submission/registering-samples) is the first step in the data submission life cycle. You **must** register samples before submitting molecular data. Please ensure that your samples are registered on the ARGO Data Platform before continuing with this step.
-
+:::note
+ [Sample Registration](/docs/submission/registering-samples) is the first step in the data submission life cycle. You **must** register samples before submitting molecular data. Please ensure that your samples are registered on the ARGO Data Platform before continuing with this step.

```

Comment:
is life cycle the right word there?
Should it be "process"?

Reply:
done